system/web/services/ModuleService.cfc

```diff
@@ -248,7 +248,8 @@ I oversee and manage ColdBox modules
 						handlersLocation 	= "handlers",
 						layoutsLocation 	= "layouts",
 						viewsLocation 		= "views",
-						modelsLocation      = "models"
+						modelsLocation      = "models",
+						includesLocation    = "includes"
```

Comment:
Is there any where else in modules that needs the `includesLocation`?  I couldn't see anywhere it was used....